src/index.html

```diff
@@ -0,0 +1,289 @@
+          <h1 class="quality-card__title">Premium quality performance</h1>
+          <p class="quality-card__body">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+        <div class="quality-card">
+          <img class="quality-card__icon" src="/image/headPhone.svg" alt="" class="flex-section__img" />
+          <h1 class="quality-card__title">Full time customer support-24/7</h1>
+          <p class="quality-card__body">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+       
+      </div>
+    </div>
+    <!-- Why should you work with us -->
+      <div class="work-section">
+        <div class="work-section-card">
+          <p class="work-section__tag">Why should you work with us</p>
+          <h1 class="work-section-card__title work-section-card__title--big">To upscale your business to the next level</h1>
+          <p class="work-section__body">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua invidunt ut labore.
+          </p>
+        </div>
+        <div class="work">
+          <div class="work__card">
+            <img class="work__icon" src="/image/arrow.svg" alt="">
+            <p class="work__body">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </div>
+          <div class="work__card">
+           <img class="work__icon" src="/image/arrow.svg" alt="">
+            <p class="work__body">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </div>
+          <div class="work__card">
+            <img class="work__icon" src="/image/arrow.svg" alt="">
+            <p class="work__body">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </div>
+         
+        </div>
+      </div>
+      <!-- Impressions -->
+      <div class="impressions">
+        <div class="impressions__card">
+          <h1 class="impressions__title">More impressions, more conversions</h1>
+          <p class="impressions__body">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua invidunt ut labore.
+          </p>
+          <button class="impressions__btn">Get Free trial</button>
+        </div>
+        <img class="impressions__image" src="/image/statistics.png" alt="" />
+      </div>
+      <!-- Logo ipsum -->
+      <div class="companyion">
+        <img class="companyion__image" src="/image/Logoipsum.png" alt="">
```

Comment:
companyion, I think you made mistake in spelling, maybe you wanted to say company